src/lib/multipageloader.cc

```diff
@@ -107,6 +107,20 @@ QNetworkReply * MyNetworkAccessManager::createRequest(Operation op, const QNetwo
 	return QNetworkAccessManager::createRequest(op, r3, outgoingData);
 }
 
+MyNetworkProxyFactory::MyNetworkProxyFactory (QNetworkProxy proxy, QList<QString> bph, ResourceObject & res):
```

Comment:
Just an oversight - I was originally using it, then removed it but missed this line. Thanks!